### ISSUE TITLE
APT repository signing key changed?

### ISSUE THREAD

**Issue by ypcs** [allinurl/goaccess#1775]
Just noticed on some servers that APT repository signing key has changed. Old key was
```
pub   rsa4096 2014-05-20 [SC]
      FA0B F24A 0F94 6367 0B88  0EAB 7424 83B7 B222 C443
uid           [ unknown] Gerardo Orellana <goaccess@prosoftcorp.com>
sub   rsa4096 2014-05-20 [E]
```
and new key (unconfirmed) seems to be
```
pub   rsa3072 2020-05-17 [SC] [expires: 2022-05-17]
      C03B48887D5E56B046715D3297BD1A0133449C3D
uid           [ unknown] Gerardo Orellana <goaccess@prosoftcorp.com>
sub   rsa3072 2020-05-17 [E] [expires: 2022-05-17]
```

Can you @allinurl confirm that this is valid change, and would it be possible to

1) publish they key fingerprint for valid key somewhere in GoAccess.io to make it easier to have at least one additional place to confirm the key validity and
2) sign the new key with the old one so that it's possible to cryptographically verify that "the person who had access to old key has confirmed that new key is OK".

I'm aware of [this note in the download page](https://goaccess.io/download)
![Screenshot from 2020-05-18 15-33-26](https://user-images.githubusercontent.com/97568/82213518-eea2c480-991c-11ea-8340-720ed3d3ef80.png)
but in case of someone breaching website(s) goaccess.io and deb.goaccess.io there is no way currently to ensure that this is legitimate change, and I'm not willing to just pull in "random key" to production machines.

**Comment by allinurl** [comment]
Thanks for posting this. Yes, I can confirm that this is a valid change. 

1. I went ahead and posted the key fingerprint on the [downloads page](https://goaccess.io/download#official-repo ) as well.
2. Unfortunately I won't be able to sign the old key as I had a drive issue with the server containing the key, thus this change.

I apologize about this sudden changes. Though, I hope this helps to confirm these changes :)

**Comment by ypcs** [comment]
Thanks for the fast reply and action, and also for awesome tool :)

Yes, this helps, while not the ideal solution, it's probably best that can be done if old keys got lost (to confirm authenticity via other channels, like GitHub, Twitter, website etc.).

I'm leaving this issue open, as I'm not sure if you want to leave this for others to notice, but feel free to close when it's right time. :)

**Comment by allinurl** [comment]
Great! I'll keep it open for a few weeks. Thank you!

**Comment by EqUaTe** [comment]
The fingerprint is there, which is great, but the file is now missing, so the instructions for Debian-based distributions are currently invalid.
It also doesn't appear that the public key has been uploaded to any keyservers? You may want to consider doing this (it's usually considered good practice).

**Comment by allinurl** [comment]
@EqUaTe not sure what you mean with the fine is missing. As per the instructions, you can download it here https://deb.goaccess.io/gnugpg.key .

**Comment by EqUaTe** [comment]
@allinurl Sorry, looks like I fat-fingered it - gnupg.key rather than gnugpg.key (OOI, why is it gnugpg.key? The product is GnuPG :-) )

The other part stands about keyservers though - then the instructions can be something like:
sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 97BD1A0133449C3D

**Comment by allinurl** [comment]
@EqUaTe Yeah not sure where the `gnugpg.key` came from. For now I'll keep it that way just because there are websites out there with instructions pointing to `gnugpg.key`.

**Comment by dregad** [comment]
May I suggest to update the [Ubuntu install procedure](http://goaccess.io/download#official-repo), to create a GoAccess-specific keyring instead of dumping the key in the default one ?

```
wget -O - https://deb.goaccess.io/gnugpg.key | 
  sudo apt-key --keyring /etc/apt/trusted.gpg.d/goaccess.gpg add -
```

**Comment by allinurl** [comment]
@dregad Updated. Thanks!
